recipes/plaster/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: pan-plaster
+  version: '{{version}}'
+
+source:
+  url: 'https://gitlab.com/treangenlab/plaster/-/archive/v{{version}}/plaster-v{{version}}.tar.gz'  
+  sha256: c43005222bbb92207a04e331f4aa3e4e72c986d94d5c766ecb6e3ceab64e9bbb                                   
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    - biopython
+    - python >=3
+    - mummer4
+    - tqdm
+
+test:
+  source_files:
+    - example/
+    - example/*
+  commands:
+    - plaster $PREFIX/bin/example/*.fasta
+
+about:
+  home: https://gitlab.com/treangenlab/plaster 
+  license: MIT
+  license_file: LICENSE
+  summary: Plaster is a program for quick linear pangenome construction.
+
+extra:
+  skip-lints:
+    - folder_and_package_name_must_match
```

Comment:
why not also rename the folder, I guess this makes it easier to find the recipe for developers

Reply:
I tried by adding the `folder:` section to `source`, but then my local build started failing for another reason... ): I'm sure there's a correct way to do it, but I'm still new to figuring out these `meta.yml`

Reply:
the bioconda folder is meant, you should rename the recipes/plaster folder :)

Reply:
Ahh understood. I have made the change. Thanks!